build.sc

```diff
@@ -188,6 +188,13 @@ trait MillMimaConfig extends mima.Mima {
     issueFilterByModule.getOrElse(this, Seq())
   }
   lazy val issueFilterByModule: Map[MillMimaConfig, Seq[ProblemFilter]] = Map(
+    main.core -> Seq(
+      // fixed from Some[T] to Option[T]
+      ProblemFilter.exclude[DirectMissingMethodProblem]("mill.define.Target.asTarget"),
+      ProblemFilter.exclude[DirectMissingMethodProblem]("mill.define.TargetImpl.asTarget"),
```

Comment:
This will make all plugins compiled with older Mill versions that use these methods crash at runtime.
I think we will need to delay this change for `0.11`